config/api.yml

```diff
@@ -1889,6 +1896,14 @@
         :identifier: service_tag
       - :name: unassign
         :identifier: service_tag
+    :orchestration_stacks_subcollection_actions:
+      :get:
+      - :name: read
+        :identifier: orchestration_stack_view
```

Comment:
Since these identifiers don't seem linked in any way to the parent services collection, you could put this configuration under the orchestration stacks section. But not a blocker - I just prefer to think of this section as configuration that's tailored for this parent collection :smile: